bin/cluster

```diff
@@ -290,6 +291,8 @@ if __name__ == '__main__':
                                help='number of infra nodes to create in cluster')
     create_parser.add_argument('-e', '--etcd', default=0, type=int,
                                help='number of external etcd hosts to create in cluster')
+    create_parser.add_argument('--env', default='prod', type=str,
```

Comment:
@detiber,

Most of our customers will be classified as 'prod' environment.  I don't think devs care much about the environment since we aren't currently using it.  That said, I'm up for suggestions.